internal/test/framework/framework.go

```diff
@@ -175,3 +177,14 @@ func (f *TestFramework) TearDown() {
 		}
 	}
 }
+
+// WriteToArtifactDir will write contents to $ARTIFACT_DIR/subDirName/filename
```

Comment:
Please mention that `subDirName=""` will result in the file being created in the artifact directory.